modules/speller/default/language.cpp

```diff
@@ -639,7 +639,7 @@ namespace aspeller {
 
     String dir1,dir2,path;
     fill_data_dir(&c, dir1, dir2);
-    String dir = find_file(path,dir1,dir2,lang,".dat");
+    const String &dir = find_file(path,dir1,dir2,lang,".dat");
```

Comment:
I don't agree with clang-tidy here.  Please leave as is.

Reply:
The current code is creating a copy of the result of find_file without any need.

find_file returns a const String & so we can just use the reference instead of copying it to a new String

Why is that's bad?

Reply:
Because in the grand scheme of things it doesn't make any real difference and `String` is better in case the return value to find_file changes to return a value rather than a reference.